### PR TITLE
xlogy

### LINKED ISSUE #21869: xlogy
Todo list: #3612

### PR DESCRIPTION
**Pull request by NiteshK84**
# PR Description 
Adds xlogy function to torch frontend

## Related Issue 

Close #21869 

## Checklist 

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by aparajith21** [comment]
Hi, thank you for contributing! There appear to be some failures on the CI. Could you please check these locally and see what shows up there. Further, it would be great if you could check off the items in the front end checklist by commenting `add_frontend_checklist` and confirm everything has been completed properly before requesting review.Thanks for taking the time and contributing!

**Comment by aparajith21** [comment]
Please close this one as it seems to be a repeat of another PR you created! Thanks

**Comment by NiteshK84** [comment]
> Please close this one as it seems to be a repeat of another PR you created! Thanks

Hey @aparajith21, 
actually this is not a clone
the other PR implements the inplace version of this function, it must be suffix with an underscore, which ever other PR you saw,
i'll fix the merge conflicts here please review and merge it
Thanks  😄

**Comment by NiteshK84** [comment]
@aparajith21, i fixed merge conflicts, 
its ready to merge.

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -1986,6 +1986,19 @@ def unique_consecutive(self, return_inverse, return_counts, dim):
     def cummax(self, dim):
         return torch_frontend.cummax(self, dim)
 
+    @with_unsupported_dtypes(
+        {
+            "2.0.1 and below": (
+                "float16",
+                "bfloat16",
```
Comment (aparajith21) [inline ivy/functional/frontends/torch/tensor.py:1993]: We seem to be having a bunch of unsupported dtype errors in the CI, could you please look into the same and add to the list here and request review once tests are passing? Thanks for contributing!

**Comment by NiteshK84** [comment]
ready!

**Pull request merged**